src/lib/DatePicker.svelte

```diff
@@ -252,10 +257,13 @@
           ><path d="M6 0l12 12-12 12z" transform="rotate(90, 12, 12)" /></svg
         >
       </div>
-      <div class="dropdown year">
+      <div class="dropdown year" tabindex="-1" bind:this={yearDiv}>
```

Comment:
I gotta check if the `tabindex="-1"` causes an unexpected outline to show up

Reply:
Seems to be working well on my end but please do have a look! 🙂